webapp-django/crashstats/settings/base.py

```diff
@@ -376,12 +346,260 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # all the migrations.
 SOUTH_TESTS_MIGRATE = False
```

Comment:
Why not all of these? 

The point of having `SOUTH_TESTS_MIGRATE` (**for example**) here in settings is so you can override them if you need to. 
Well, it's kinda nice to have them written all down in one place too I guess.